src/main/java/org/opensearch/ad/AnomalyDetectorJobRunner.java

```diff
@@ -1,698 +1,697 @@
-// boolean releaseLock,
-// String taskState
-// ) {
-// String detectorId = jobParameter.getName();
-// try {
-// IntervalTimeConfiguration windowDelay = (IntervalTimeConfiguration) jobParameter.getWindowDelay();
-// Instant dataStartTime = detectionStartTime.minus(windowDelay.getInterval(), windowDelay.getUnit());
-// Instant dataEndTime = executionStartTime.minus(windowDelay.getInterval(), windowDelay.getUnit());
-// UserIdentity user = jobParameter.getUser();
-//
-// AnomalyResult anomalyResult = new AnomalyResult(
-// detectorId,
-// null, // no task id
-// new ArrayList<FeatureData>(),
-// dataStartTime,
-// dataEndTime,
-// executionStartTime,
-// Instant.now(),
-// errorMessage,
-// null, // single-stream detectors have no entity
-// user,
-// anomalyDetectionIndices.getSchemaVersion(ADIndex.RESULT),
-// null // no model id
-// );
-// String resultIndex = jobParameter.getResultIndex();
-// if (resultIndex != null && !anomalyDetectionIndices.doesIndexExist(resultIndex)) {
-// // Set result index as null, will write exception to default result index.
-// anomalyResultHandler.index(anomalyResult, detectorId, null);
-// } else {
-// anomalyResultHandler.index(anomalyResult, detectorId, resultIndex);
-// }
-//
-// updateLatestRealtimeTask(detectorId, taskState, null, null, errorMessage);
-// } catch (Exception e) {
-// log.error("Failed to index anomaly result for " + detectorId, e);
-// } finally {
-// if (releaseLock) {
-// releaseLock(jobParameter, lockService, lock);
-// }
-// }
-// }
-//
-// private void updateLatestRealtimeTask(
-// String detectorId,
-// String taskState,
-// Long rcfTotalUpdates,
-// Long detectorIntervalInMinutes,
-// String error
-// ) {
-// // Don't need info as this will be printed repeatedly in each interval
-// adTaskManager
-// .updateLatestRealtimeTaskOnCoordinatingNode(
-// detectorId,
-// taskState,
-// rcfTotalUpdates,
-// detectorIntervalInMinutes,
-// error,
-// ActionListener.wrap(r -> {
-// if (r != null) {
-// log.debug("Updated latest realtime task successfully for detector {}, taskState: {}", detectorId, taskState);
-// }
-// }, e -> {
-// if ((e instanceof ResourceNotFoundException) && e.getMessage().contains(CAN_NOT_FIND_LATEST_TASK)) {
-// // Clear realtime task cache, will recreate AD task in next run, check AnomalyResultTransportAction.
-// log.error("Can't find latest realtime task of detector " + detectorId);
-// adTaskManager.removeRealtimeTaskCache(detectorId);
-// } else {
-// log.error("Failed to update latest realtime task for detector " + detectorId, e);
-// }
-// })
-// );
-// }
-//
-// private void releaseLock(AnomalyDetectorJob jobParameter, LockService lockService, LockModel lock) {
-// lockService
-// .release(
-// lock,
-// ActionListener
-// .wrap(
-// released -> { log.info("Released lock for AD job {}", jobParameter.getName()); },
-// exception -> { log.error("Failed to release lock for AD job: " + jobParameter.getName(), exception); }
-// )
-// );
-// }
-// }
+/*
```

Comment:
Holy Large Diff, Batman!

It's really difficult to evaluate just the code that changed when it looks like an entirely new file.  Is there a commit somewhere that separates out the uncommenting in one commit, and the code changes in a second commit, or any other way of comparing this new file to the original version of the file (such as an earlier release)?

For future reference for the whole team, using block comments with a hyphen preserves formatting between them, so
```java
    /*-
    original code(String whatever) {
        noChangesToIndentation();
        makesDiffsEasier();
    }
    */
```

Reply:
I just uncommented the AnomalyDetectorJobRunner and I haven't done any change in that.

Reply:
ACK the suggestion